### DISCUSSION TITLE
如何让图片的原生markdown语法支持放大

### DISCUSSION THREAD

**Discussion by WeiXinao** [xaoxuu/hexo-theme-stellar#204] [Q&A] [answered]
为什么使用‘! [] ()’语法嵌入的图片无法用fancybox放大

**Comment by xaoxuu** [comment, marked as answer]
目前不支持，请使用功能更强大的 `{% image xxx %}` 语法。

**Reply by WeiXinao** [reply]
后续会支持么？

**Reply by xaoxuu** [reply]
暂时没有计划

**Reply by WeiXinao** [reply]
![image](https://user-images.githubusercontent.com/65220113/206387476-4a743f1f-8fbf-42c4-bec9-9c9475381086.png)
好的，另外我想请问一下，你博客里面的图片资源是如何获取的，我目前还没有获取博客图片的好途径

**Reply by xaoxuu** [reply]
flaticon.com

**Reply by WeiXinao** [reply]
![image](https://user-images.githubusercontent.com/65220113/206860777-ea87b363-bd5c-43cc-b9b1-ceb17f7a723d.png)
![image](https://user-images.githubusercontent.com/65220113/206860852-4458137d-477e-4532-81b0-51beff48dab4.png)
为什么我这个博客鼠标放在标题上没有触发锚点
